tfrecorder/client.py

```diff
@@ -36,7 +36,7 @@
 # TODO(mikebernico) Add test for only one split_key.
 def _validate_data(df: pd.DataFrame,
                    schema_map: Dict[str, collections.namedtuple]):
-  """ Verifies required image csv columns exist in data."""
+  """Verifies required image csv columns exist in data."""
```

Comment:
Consider rewording: "Verifies data is consistent with schema."""

Reply:
Ahh, yes, I guess this function has changed since the docstring was written.